src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -377,15 +377,17 @@ class ShenandoahHeap : public CollectedHeap {
 
   bool _upgraded_to_full;
 
+  bool _has_evacuation_reserve_quantities;
```

Comment:
Could we have a description of this new field in the header?

Reply:
Comment has been added.